doc/source/whatsnew/v1.2.0.rst

```diff
@@ -109,6 +109,25 @@ Beginning with this version, the default is now to use the more accurate parser
 ``floating_precision="legacy"`` to use the legacy parser. The change to using the higher precision
 parser by default should have no impact on performance. (:issue:`17154`)
 
+.. _whatsnew_120.index_name_preservation:
+
+Index/column name preservation when aggregating
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When aggregating using :meth:`concat` or the :class:`DataFrame` constructor, Pandas
```

Comment:
can you add the issue reference here.

Also I would add the same text here in the docs, maybe in the reshaping section, in a note

Reply:
yep, i ended up adding it as a note in the "merge, join, concatenate, and compare" section.